### DISCUSSION TITLE
The units of the solutions in the "PreprocessingVariationalEquationOrbitFit" program

### DISCUSSION THREAD

**Discussion by ReubenLi1999** [groops-devs/groops#235] [Q&A] [answered]
Hello developers,

I'm currently working with the "PreprocessingVariationalEquationOrbitFit" program and have encountered a puzzling aspect related to the values in the solution file. I would appreciate some clarification from the developers to better understand the units of these values.

What I am doing now is using "PreprocessingVariationalEquationOrbitFit" program to generate integrated dynamic orbits of GRACE-FO satellites on a daily basis. I also want to get ACC calibration results and initial orbit states. Now, these parameters are estimated as a daily constant. However, I am puzzled about the units of these parameters.

If it helps, here's my results:
![image](https://github.com/groops-devs/groops/assets/49586424/2bd6bd38-0620-4550-91f3-6285f5a48503)
![image](https://github.com/groops-devs/groops/assets/49586424/81064a60-8e14-40ad-b4ed-3f05c8217533)

It appears that the initial position in the above picture is being treated incrementally from the last iteration, contrary to the expectation of a full initial position.

I appreciate your time and assistance in helping me gain clarity on this matter.

Cheers,

Haosi

**Comment by tmayerguerr** [comment, marked as answer]
Hi Haosi,

You are right: The program reads the `inputfileVariational` and estimates the fit parameters relative to this orbit (units m and m/s). We have never used these parameters before and GROOPS is not designed to calculate the absolute state vector.

regards
Torsten

**Reply by ReubenLi1999** [reply]
Thank you Torsten.

**Reply by ReubenLi1999** [reply]
One more thing. Could you kindly explain that there is any regularization applied in the estimation of these parameters? If so, what kind of regularization is applied? Thank you!

**Reply by tmayerguerr** [reply]
No regularization is applied

**Reply by ReubenLi1999** [reply]
@tmayerguerr OK. Thank you very much.
